packages/frontapp/src/app/components/Header/index.tsx

```diff
@@ -37,10 +37,11 @@ const WrapperForCenter = styled.div`
 const WrapperHeader = styled.header`
   max-width: 16.5rem;
   width: 100%;
-  margin-bottom: 2rem;
+  margin-bottom: 3.5rem;
   @media (max-width: ${dimensions.tablet_width}) {
     max-width: none;
     padding: 0 20px;
+    margin-bottom: 2.5rem;
```

Comment:
I would recommend to use variables from dimensions file.

Reply:
Make sense